schema/buildings/defs.yaml

```diff
@@ -118,3 +116,22 @@ shapeContainer:
         The height of the building roof in meters. This represents the distance
         from the base of the roof to the highest point of the roof.
       type: number
+  propertyContainers:
+    shapeContainer:
+      title: shape
+      description: Properties of the buildings shape, such as height or roof type.
+      properties:
+        height: {"$ref" : "#/$defs/propertyDefinitions/height"}
+        is_underground: {"$ref" : "#/$defs/propertyDefinitions/is_underground"}
+        num_floors: {"$ref" : "#/$defs/propertyDefinitions/num_floors"}
+        num_floors_underground: {"$ref" : "#/$defs/propertyDefinitions/num_floors_underground"}
+        min_height: {"$ref" : "#/$defs/propertyDefinitions/min_height"}
+        min_floor: {"$ref" : "#/$defs/propertyDefinitions/min_floor"}
+        facade_color: {"$ref" : "#/$defs/propertyDefinitions/facade_color"}
+        facade_material: {"$ref" : "#/$defs/propertyDefinitions/facade_material"}
+        roof_material: {"$ref" : "#/$defs/propertyDefinitions/roof_material"}
+        roof_shape: {"$ref" : "#/$defs/propertyDefinitions/roof_shape"}
+        roof_direction: {"$ref" : "#/$defs/propertyDefinitions/roof_direction"}
+        roof_orientation: {"$ref" : "#/$defs/propertyDefinitions/roof_orientation"}
+        roof_color: {"$ref" : "#/$defs/propertyDefinitions/roof_color"}
+        roof_height: {"$ref" : "#/$defs/propertyDefinitions/roof_height"}
```

Comment:
Can't you just put all the definitions here rather than have them by reference? They're not used anywhere else right?

Reply:
Yep, done.